content/en/tracing/other_telemetry/connect_logs_and_traces/php.md

```diff
@@ -24,8 +24,95 @@ further_reading:
+- [Use placeholders in your message](#use-placeholders-in-your-message)
+- [Append the trace correlation identifiers to your message](#append-the-trace-correlation-identifiers-to-your-message)
+- [Add the trace correlation identifiers to the log context](#add-the-trace-correlation-identifiers-to-the-log-context)
+
+#### Use placeholders in your message {#use-placeholders-in-your-message}
+
+You can use placeholders in your message to automatically inject trace correlation identifiers into your logs. The PHP tracer supports the following placeholders:
+- `%dd.trace_id%`: the trace ID
+- `%dd.span_id%`: the span ID
+- `%dd.service%`: the service name
+- `%dd.version%`: the service version
+- `%dd.env%`: the service environment
+- `%status%`: the severity level of the log event
+
+Placeholders are case-sensitive and must be enclosed in `%` characters.
+
+For example, if you are using the [Monolog][4] library in a Laravel application, you can configure the injection into a log message as follows:
+
+```php
+use Illuminate\Support\Facades\Log;
+# ...
+Log::info('Hello, World! [%dd.trace_id% %dd.span_id% %status%]');
+```
+
+The PHP tracer replaces the placeholders with the corresponding values. For example, the logged message above could be transformed into:
+
+```
+[2022-12-09 16:02:42] production.INFO: Hello, World! [dd.trace_id="1234567890abcdef" dd.span_id="1234567890abcdef" status="info"]
+```
+
+**Note**: the brackets are mandatory if you plan on using the default parsing rules provided in the PHP [log pipeline][7]. If you are using a custom parsing rule, you can omit the brackets if needed.
+
+#### Append the trace correlation identifiers to your message {#append-the-trace-correlation-identifiers-to-your-message}
+
+You can append the trace correlation identifiers to your message by using the `DD_TRACE_APPEND_TRACE_IDS_TO_LOGS` environment variable. The PHP tracer appends the available trace correlation identifiers at the end of your message enclosed in brackets, **modifying the original message**.
+
+For example, if you are using the [Monolog][4] library in a Laravel application as follows:
+
+```php
+use Illuminate\Support\Facades\Log;
+# ...
+Log::debug('Hello, World!');
+```
+
+The PHP tracer appends the available trace correlation identifiers to the log message. The logged message above could be transformed into:
+
+```
+[2022-12-09 16:02:42] production.DEBUG: Hello, World! [dd.trace_id="1234567890abcdef" dd.span_id="1234567890abcdef" dd.service="laravel" dd.version="8.0.0" dd.env="production" status="debug"]
+```
+
+**Note**: If there is a placeholder in your message or if a trace ID is already present in the message, that takes precedence and the PHP tracer does **not** further modify the message.
+
+#### Add the trace correlation identifiers to the log context {#add-the-trace-correlation-identifiers-to-the-log-context}
+
+The default behavior of the PHP tracer is to add the trace correlation identifiers to the log context.
```

Comment:
If this is the default, should it be the first solution documented?

Reply:
Disclaimer: I don't mind putting it first

It is the default behavior, but it is not the prioritized one*. Placeholders will take precedence over it.

Reply:
Ok which one do we prefer? This one means less work for customers, right? If that's the case, that's the first one that should be listed

Reply:
Yep, that's the case. I'll do the change :+1: